pkg/kubectl/cmd/drain_test.go

```diff
@@ -244,7 +246,18 @@ func TestDrain(t *testing.T) {
 			Namespace:         "default",
 			CreationTimestamp: metav1.Time{Time: time.Now()},
 			Labels:            labels,
+			SelfLink:          testapi.Default.SelfLink("replicationcontrollers", "rc"),
```

Comment:
Although it doesn't matter for this test, SelfLink should link to the pod itself, not its controller.